projects/laji/src/app/shared-modules/spreadsheet/service/mapping.service.ts

```diff
@@ -18,13 +18,15 @@ export enum SpecialTypes {
   dateTime = 'dateTime',
   date = 'date',
   time = 'time',
+  keywords = 'keywords'
 }
 
 @Injectable()
 export class MappingService {
 
   public static readonly mergeKey = '_merge_';
   public static readonly valueSplitter = ';';
+  public static readonly vihkoFormsSplitters = [';'];
```

Comment:
This seems to be no longer used but it could be used in the `mapKeywords`. Maybe rename this to `keywordSplitters = [';', ',']` and use that in the mapKeywords method